ui/src/app/utils/runtime-utils.tsx

```diff
@@ -395,6 +474,9 @@ export const useCustomRuntime = (currentWorkspaceNamespace):
                 pollAbortSignal: aborter.signal,
                 overallTimeout: 1000 * 60 // The switch to a non running status should occur quickly
               });
+            } else if (runtime.status === RuntimeStatus.Deleted && enablePD && pdIncreased) {
```

Comment:
nit: `enablePD` seems unecessary here - we can't get to this point in the code unless `enablePD` was active